internal/model/model.go

```diff
@@ -1222,6 +1218,7 @@ nextSub:
 		TempNamer:     defTempNamer,
 		TempLifetime:  time.Duration(m.cfg.Options().KeepTemporariesH) * time.Hour,
 		CurrentFiler:  cFiler{m, folder},
+		MtimeRepo:     db.NewVirtualMtimeRepo(m.db, folderCfg.ID),
```

Comment:
If you can come up with a better name then I'm all for it.  This one seemed the most accurate and descriptive without being unnecessarily verbose.